lib/SystemStatistics.php

```diff
@@ -77,13 +77,27 @@ protected function getMemoryUsage() {
 		}
 		//If FreeBSD is used and exec()-usage is allowed
 		if (PHP_OS === 'FreeBSD' && \OC_Helper::is_function_enabled('exec')) {
+			//Read Swap usage:
+			exec("/usr/sbin/swapinfo",$return,$status);
+			if ($status===0 && count($return) > 1) {
+				$line = preg_split("/[\s]+/", $return[1]);			
+				if(count($line) > 3) {
+					$swapTotal = (int) $line[3];
+					$swapFree = $swapTotal- (int) $line[2];
+				}
+			}
+			unset($status);
```

Comment:
Why is there two `unset()`? Those should not be needed and also they would cause the if case in line 93 to never be true.

Reply:
I want to use them again in line 93 (it's the output of the `exec()` command). And I wanted to avoid that in case it fails it keeps the previous values and does strange things. I could have used new variables in line 93 if that is preferred.

Reply:
A missed that - then it's fine 👍